www/src/components/header/Header.tsx

```diff
@@ -59,7 +59,6 @@ function Header({ siteTitle, showMinimizedTitle }: IHeaderProps) {
             positionRef={target}
             isOpen={isOpen}
             onClose={close}
-            onEscapeKey={close}
```

Comment:
Was this change accidentally included? Or is this not needed because it already handles the escape key?

Reply:
@bradenmacdonald `ModalPopup` doesn't accept any param called `onEscapeKey`. `FocusOn` component has `onEscapeKey` but [we pass](https://github.com/open-craft/paragon/blob/94b8eb62e46c9e4e641ac3ea5a5c4f165cd499bb/src/Modal/ModalPopup.tsx#L101) `onClose` to it.